src/olympia/hero/models.py

```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+from django.db import models
+
+from olympia.amo.models import ModelBase
+from olympia.discovery.models import DiscoveryItem
+
+
+class PrimaryHero(ModelBase):
+    image = models.CharField(max_length=255)
+    background_color = models.CharField(max_length=7)
+    enabled = models.BooleanField(db_index=True, null=False, default=False,)
+    disco_addon = models.OneToOneField(DiscoveryItem, on_delete=models.CASCADE)
+
```

Comment:
I don't see it listed in the issue, nor in the requirements doc, but I recall a discussion about the ability to pin a single add-on to always use in the Primary Hero area, rather than allowing random images to cycle. If we are still planning on implementing that, this seems like the place we'd store that piece of data.

@jvillalobos Do we still plan to implement the ability to pin an individual add-on in the Primary Hero area?

Reply:
It got dropped as a requirement.  You could still do that by setting the other PrimaryHero objects to `enabled=False` though.

Reply:
Pinning was dropped. Like Andrew said, it's enough to have a single extension in the homepage featured group.